draft-ietf-ppm-dap.md

```diff
@@ -2459,23 +2509,12 @@ designed to allow implementations to reduce operational costs in certain cases.
 In general, the Aggregators are required to keep state for tasks and all valid
 reports for as long as collection requests can be made for them. However, it is
 not necessary to store the complete reports. Each Aggregator only needs to store
-an aggregate share for each possible batch interval (for time-interval) or batch
-ID (for leader-selected), along with a flag indicating whether the aggregate
-share has been collected. This is due to the requirement that in the
+an aggregate share for each batch, along with a flag indicating whether the
+aggregate share has been collected. This is due to the requirement that in the
```

Comment:
I believe the old text still applies: a batch interval is a sequence of time windows; and the batch interval is not known at aggregation time.

Reply:
Hmm, I think the old text was slightly inaccurate for the time-interval case: aggregators don't have to store an aggregate share for each possible a batch interval (i.e., sequence of time windows) -- an aggregator needs to store an aggregate share for each _time window_.

I had felt that distinguishing between the batch modes was not so useful here; I restored the text, modified for accuracy.